PathPlanning/DStarLite/d_star_lite.py

```diff
@@ -62,32 +65,46 @@ def __init__(self, ox: list, oy: list):
         self.y_max = int(abs(max(oy) - self.y_min_world))
         self.obstacles = [Node(x - self.x_min_world, y - self.y_min_world)
                           for x, y in zip(ox, oy)]
+        self.obstacles_xy = np.array(
+            [[obstacle.x, obstacle.y] for obstacle in self.obstacles]
+        )
         self.start = Node(0, 0)
         self.goal = Node(0, 0)
         self.U = list()  # type: ignore
         self.km = 0.0
         self.kold = 0.0
-        self.rhs = list()  # type: ignore
-        self.g = list()  # type: ignore
+        self.rhs = np.full(
+            (self.x_max, self.y_max),
+            float("inf")  # type: ignore
+        )
```

Comment:
I think introducing a new function to initialize a grid map is better like:

```
def init2dGrid():
       return np.full(self.x_max, self.y_max), float("inf")
```

Reply:
I used an already existing function called create_grid. Thanks for the improvement.